feat(rtl): right-to-left batch 1 - FRONT-3565

emeryro opened the pull request:
- card
- content-item
- list-illustration
- inpage-navigation
- link
- menu
- spinner
- tabs
- tag

Notes
- there are a few elements that will be handled in another batch (for instance description list). So display may not be fully operational yet (would need a final review when all batch are done)
- the javascript for tabs has to be updated to fix mobile display. For now I focused on the css, we will handle the few tricky components after

papegaill commented on src/implementations/vanilla/components/tabs/tabs.js:
```diff
@@ -190,7 +190,7 @@ export class Tabs {
     }
     this.list.style.transitionDuration = '0.4s';
-    this.list.style.transform = `translate3d(-${newOffset}px, 0px, 0px)`;
+    this.list.style.transform = `translate3d(${newOffset}px, 0px, 0px)`;
```
this breaks the navigation for the non-rtl tabs in mobile, the pagination is in reverse, I won't apply this change now?

emeryro replied:
yes, I did it for testing and forgot to remove... I'll update

papegaill reviewed (changes requested):
Really nice, good result already with only little extra css!
I just noticed 2 small things:
<img width="288" alt="Screenshot 2022-03-15 at 18 18 19" src="https://user-images.githubusercontent.com/1289480/158555931-d2fc43ad-779d-4c71-9a7f-3302f04b3442.png">
For the inpage, active background/border should be reversed too, but it may already be too much detail..
<img width="186" alt="Screenshot 2022-03-15 at 18 20 32" src="https://user-images.githubusercontent.com/1289480/158556269-269e76b7-e95f-4bfe-92e1-937192a1a386.png">
for tabs, the spacing of the icon on the more button should be corrected, to see if this will be taken into account in another batch with the js fixes?

emeryro commented:
I updated the inpage nav, to have something working well in both cases, and reverted the wrong js update.
For the button, it is indeed handled in another PR
